bucket/insomnium.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "0.2.3-a",
+    "description": "The Insomnia HTTP and GraphQL client with online functionality removed",
+    "homepage": "https://github.com/ArchGPT/insomnium",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ArchGPT/insomnium/releases/download/core%400.2.3-a/insomnium-0.2.3-a-full.nupkg",
```

Comment:
This version stores settings in `AppData\Roaming\Insomnium`, any reason why we can't use the portable release and persist settings instead? https://github.com/ArchGPT/insomnium/releases/download/core%400.2.3-a/Insomnium.Core-0.2.3-a-portable.exe

Reply:
Looks like the portable version saves settings in the same place so no need.